en/docs/remote-apps/connect-remote-applications.md

```diff
@@ -0,0 +1,118 @@
+
+2. To link the Ballerina application to Choreo via the application secret, open a new file in a text editor and add the Ballerina configurations you copied in Step 1. Then save it as `Config.toml` in the `<BALLERINA_PROJECT>`.
+
+3. To enable the Ballerina application to publish observability statistics, save a file named `Ballerina.toml` with the following content in the `<BALLERINA_PROJECT>`.
+
+    ```ballerina
+    [build-options]
+    observabilityIncluded = true
+    ```
+   
+4. For the Ballerina application to import the `ballerinax/choreo` module, open the `.bal` file (in this example, `hello_service.bal`) and at the following line at the top.
+
+    ```ballerina
+    import ballerinax/choreo as _;
+    ```
+   
+Now you have completed setting up your Ballerina project. It will connect to Choreo once you build and run it.
+
+## Step 3: Build the Ballerina project
+
+In this step, you will build and run your Ballerina project. 
+
+1. To build the Ballerina project, open your terminal, navigate to `<BALLERINA_PROJECT>`, and issue the following command.
+
+    ```bash
+    bal build
+    ```
+   
+    This creates a `JAR` in the `<BALLERINA_PROJECT>/target/bin` sub directory.
+   
+2. To run the Ballerina application, issue the following command from the `<BALLERINA_PROJECT>`.
+
+    ```bash
+    bal run target/bin/<JAR_NAME>.jar
+    ```
+   
+    e.g., `bal run target/bin/HelloService.jar`
+    
+    Logs similar to the following appear in the terminal.
+    
+    ```bash
+    ballerina: initializing connection with observability backend periscope.choreo.dev:443
+    ballerina: visit http://console.choreo.dev/observe/app/91e9750f-e2e1-11eb-90ce-de6d84510939/dd9780a8-e2e2-11eb-90ce-de6d84510939 to access observability data
+    ballerina: started publishing metrics to Choreo
+    [ballerina/http] started HTTP/WS listener 0.0.0.0:9090
+    ballerina: started publishing traces to Choreo
+    ```
+   
+    These logs indicate that your Ballerina application is connected to Choreo and that it is publishing data to it. Now you can invoke it and generate some observability data.
+   
+## Step 4: Invoke the Ballerina application
+
+To invoke your Ballerina application, send a few requests.
+
+In this example, you can invoke the `HelloService` application via the following cURL command.
+
+```bash
+curl http://localhost:9090/hello/sayHello
+```
+
+After sending a few requests, you can proceed to the next step to view the observability statistics generated for your Ballerina application.
+
+## Step 5: View observability statistics
+
+To view the observability statistics generated for your remote Ballerina application, click on the link provided for observability data in the startup logs of your remote application. Ah shown in the startup logs generated in [Step 3](step-3-build-the-ballerina-project), the link in this example is `http://console.choreo.dev/observe/app/91e9750f-e2e1-11eb-90ce-de6d84510939/dd9780a8-e2e2-11eb-90ce-de6d84510939`.
```

Comment:
Better to remove the term "on" in "click on the link provided for observability data......."

"Ah" seems to be a typo. Should be changed to "As".

Reply:
Corrected